index.html

```diff
@@ -2970,4 +2998,4 @@ <h2>A project for first-time contributions</h2>
     </div>
     <script type="module" src="assets/script.js"></script>
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
💡: you'll probably see this here and there as you grow in your career. If you're not sure why this showed up, it's because different file systems (operating systems) handle how a file is ended differently. 😅 On my team, we have Mac users and Windows users and we're _constantly_ seeing these.

It's not anything you did, just something your OS did when you saved your changes to the file. :) 

If you use Git by command line, there are some tools that can help you see if a change like this is being added to your commit such as `git add -p` (this means "add things... but in ***p***arts")... but otherwise on my team we mostly just ignore it. :)

Reply:
Alright @malakada Thanks for clarifying!